Rakefile

```diff
@@ -112,29 +112,21 @@ bundle exec fastlane run configure_apply force:true
     end
 
     task :install do
-      fold('install.cocoapds') do
+      fold('install.cocoapods') do
         pod %w[install]
       rescue StandardError
-        puts "Attempting to fix Gutenberg-Mobile local podspecs failing to install — since that is one of the most common reason for `pod install` to fail — then retrying…\n\n"
-        Rake::Task['dependencies:pod:fix_gbm_pods'].invoke
+        puts "`pod install` failed. Will attempt to update the Gutenberg-Mobile XCFramework — a common reason for the failure — then retrying…\n\n"
```

Comment:
"a common reason for the failure..." with that I mean, this happens to me from time to time when switching between release and main branch.